### ISSUE TITLE
Can't install 1.16.1

### ISSUE THREAD

**Issue by ndarilek** [YunoHost-Apps/gitea_ynh#40]
**How to post a meaningful bug report**
1. *Read this whole template first.*
2. *Determine if you are on the right place:*
   - *If you were performing an action on the app from the webadmin or the CLI (install, update, backup, restore, change_url...), you are on the right place!*
   - *Otherwise, the issue may be due to the app itself. Refer to its documentation or repository for help.*
   - *When in doubt, post here and we will figure it out together.*
3. *Delete the italic comments as you write over them below, and remove this guide.*
--- 

### Describe the bug

Tried installing the testing version of 1.16.1, and get the following error:

```
Info: [################++..] > Removing fail2ban configuration...                                                                    
Info: '/etc/fail2ban/jail.d/gitea.conf' wasn't deleted because it doesn't exist.                                                     
Info: '/etc/fail2ban/filter.d/gitea.conf' wasn't deleted because it doesn't exist.                                                   
```

[Here's the full log](https://paste.yunohost.org/raw/jinogewabi).

### Context

- Hardware: VPS
- YunoHost version: x.x.x
- I have access to my server: SSH/web
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: no
  - If yes, please explain:
- Using, or trying to install package version/branch:
- If upgrading, current package version: *can be found in the admin, or with `yunohost app info $app_id`*

### Steps to reproduce

- *If you performed a command from the CLI, the command itself is enough. For example:*
    ```sh
    sudo yunohost app install the_app
    ```
- *If you used the webadmin, please perform the equivalent command from the CLI first.*
- *If the error occurs in your browser, explain what you did:*
   1. *Go to '...'*
   2. *Click on '...'*
   3. *Scroll down to '...'*
   4. *See error*

```sh
$ yunohost app install https://github.com/YunoHost-Apps/gitea_ynh/tree/testing 
```

Fresh installation as of this morning.

### Expected behavior

*A clear and concise description of what you expected to happen. You can remove this section if the command above is enough to understand your intent.*

The app installs.

### Logs

*When an operation fails, YunoHost provides a simple way to share the logs.*
- *In the webadmin, the error message contains a link to the relevant log page. On that page, you will be able to 'Share with Yunopaste'. If you missed it, the logs of previous operations are also available under Tools > Logs.*
- *In command line, the command to share the logs is displayed at the end of the operation and looks like `yunohost log display [log name] --share`. If you missed it, you can find the log ID of a previous operation using `yunohost log list`.*

*After sharing the log, please copypaste directly the link provided by YunoHost (to help readability, no need to copypaste the entire content of the log here, just the link is enough...)*

*If applicable and useful, add screenshots to help explain your problem.*
https://paste.yunohost.org/raw/jinogewabi

**Comment by Josue-T** [comment]
Hello I've fixed the issue with the commit 9d7d717. Could you try again ?

**Comment by ndarilek** [comment]
Confirmed working. Thanks!
